README.md

```diff
@@ -519,6 +521,11 @@ These settings can be changed at any time.
   Bottlerocket enables the `json-file`, `awslogs`, and `none` drivers by default.
 * `settings.ecs.loglevel`: The level of verbosity for the ECS agent's logs.
   Supported values are `debug`, `info`, `warn`, `error`, and `crit`, and the default is `info`.
+* `settings.ecs.metadata-service-rps`: The steady state rate limit of the throttling configurations set for the task metadata service
+* `settings.ecs.metadata-service-burst`: The burst rate limit of the throttling configurations set for the task metadata service
+* `settings.ecs.reserved-memory`: The amount of memory, in MB, reserved for critical system processes
+* `settings.ecs.task-cleanup-wait-duration`: Time to wait before the tasks resources are removed after they are stopped.
```

Comment:
Do task resources include something other than containers?

Reply:
I'll change `resources` to `containers`, to prevent confusions.